src/storybook/components/doc-footer/doc-footer.jsx

```diff
@@ -19,3 +19,5 @@ export const DocFooter = () => (
     </div>
   </div>
 );
+
+export default DocFooter;
```

Comment:
I am thinking we will not export this one. it's only a link to our form

Reply:
Removed from `published-storybook-components`